lib/liquid/variable_lookup.rb

```diff
@@ -31,6 +31,21 @@ def initialize(markup)
       end
     end
 
+    def format
+      output = name.to_s
+      @lookups.each_index do |i|
+        output << case lookups[i]
+                  when String
+                    ".#{lookups[i]}"
+                  when VariableLookup
+                    "[#{lookups[i].format}]"
+                  else
+                    ".#{lookups[i]}"
+        end
```

Comment:
The `end` would look better aligned with the `case`

Reply:
I need to fix up my IDE formatter it better match rubocop as I'm pretty sure this failed a test it had.

Reply:
https://github.com/Shopify/ruby-style-guide#formatting

Reply:
@pushrax That is a private repository. Please post the relevant text from the guide directly here..

Reply:
@ashmaroli It is the same as this link https://shopify.github.io/ruby-style-guide/